h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -317,6 +317,23 @@ public String name(int i) {
   public Key<Vec>[] keys() { return _keys; }
   public Iterable<Key<Vec>> keysList() { return Arrays.asList(_keys); }
 
+  /**
+   * Returns all keys related to this frame (columns, rollup stats, chunks). Does not include the frame's keys itself.
+   *
+   * @return A {@link Set} of all keys belonging to this Frame
+   */
+  public Set<Key> allKeys() {
```

Comment:
this method doesn't seem to be used - can we remove it?

Reply:
Yes